assets/sass/patterns/molecules/main-menu.scss

```diff
@@ -5,47 +5,58 @@
 }
 
 .main-menu__title {
-  @include section-title-typeg();
   @include nospace();
   @include padding(5, "bottom");
+  @include section-title-typeg();
   text-transform: uppercase;
 }
 
 .main-menu__list {
   list-style: none;
+  @include margin(auto, "left");
+  @include margin(auto, "right");
   @include nospace();
-  margin-left: auto;
-  margin-right: auto;
+  @include padding(24, "left");
+
+  & :nth-child(5) {
```

Comment:
Do not hardcode the elements with borders, now they are 5th and 8th children but in future it could be changed. You can extend the yaml files with additinal property, for example you can add boolean key hasLine and if it is true, to add additional class in the mustache template, check "hasProfile" property for reference.

Reply:
I removed the hardcoding of border elements and added a boolean key